include/ur_api.h

```diff
@@ -6600,6 +6600,10 @@ typedef struct ur_exp_interop_mem_handle_t_ *ur_exp_interop_mem_handle_t;
 /// @brief Handle of interop semaphore
 typedef struct ur_exp_interop_semaphore_handle_t_ *ur_exp_interop_semaphore_handle_t;
 
+///////////////////////////////////////////////////////////////////////////////
+/// @brief Handle of file descriptor
+typedef struct ur_exp_file_descriptor_handle_t_ *ur_exp_file_descriptor_handle_t;
```

Comment:
so by defining it as handle, each adapter is free to defining as it sees fit, correct?

Reply:
Is that the intent?  A file handle doesn't really vary per adapter, it varies per host OS.